analyze/action.yml

```
@@ -67,7 +67,7 @@ inputs:
   matrix:
     default: ${{ toJson(matrix) }}
   expect-error:
-    description: For PR checks. If true, the Action will not set Actions status to failure.
+    description: "[Internal] If true, the Action will not set Actions status to failure."
```

If this is an error outside of integration testing, then specify that in the docs. Maybe:

```suggestion
    description: "[Internal] It is an error to use this input outside of integration testing of the codeql-action."
```

(Unfortunate that actions does not support hidden inputs.)